src/help.cpp

```diff
@@ -264,8 +264,9 @@ void add_options() {
   /*Encrypt table inplace encryption */
   opt = newOption(Option::INT, Option::ALTER_TABLE_ENCRYPTION_INPLACE,
                   "alter-table-encrypt-inplace");
-  opt->help = "Alter table set Encryption inplace.\nOnly available for ps";
-  opt->setInt(10);
```

Comment:
no need to set it 0 here. Set it 0 if encryption type = "ORACLE"

Reply:
As discussed, this option was removed from pstress